packages/web/src/pages/rewards-page/components/modals/ChallengeRewardsModal/types.ts

```diff
@@ -0,0 +1,42 @@
+import { ReactNode } from 'react'
+
+import { ChallengeName, OptimisticUserChallenge } from '@audius/common/models'
+
+export type BaseChallengeContentProps = {
+  challenge?: OptimisticUserChallenge
+  onNavigateAway: () => void
+  errorContent?: ReactNode
+}
+
+export type AudioMatchingChallengeProps = BaseChallengeContentProps & {
+  challengeName:
+    | ChallengeName.AudioMatchingBuy
+    | ChallengeName.AudioMatchingSell
+}
+
+export type ListenStreakChallengeProps = BaseChallengeContentProps & {
+  challengeName: ChallengeName.ListenStreakEndless
+}
+
+export type DefaultChallengeProps = BaseChallengeContentProps & {
+  challengeName: Exclude<
+    ChallengeName,
```

Comment:
super small nit, but i think if this is higher up in the file it will make more people to remember to add to the exclusion. non-fact based, just vibes